### PR TITLE
Implemented PromptDAO, most of the login/signup logic

### PR DESCRIPTION
**Pull request by chowdberry**
- This branch was mainly based around implementing basic login/signup use case, the starting "landing page" and implementing the prompt DAO as well as any methods associated with it that are required.

** Some work still needs to be done and will be updated soon, but this contains most of the minimum necessary for the application to run.

### CODE REVIEW

**Review by NimratGrewal — APPROVED** [review]

**Comment by hayatariq** [comment]
just remove creation time! also method DeleteResponse does not follow java naming conventions, better to name it deleteResponse. Other than that it looks good!

**Comment by hayatariq** [comment]
Please include getPromptByID and getResponses! would help me out, thanks

File: src/data_access/PromptDataAccessObject.java
Comment (hayatariq) [inline src/data_access/PromptDataAccessObject.java:1]: No need for creation Time I think!

File: src/interface_adapter/login/LoginPresenter.java
Comment (hayatariq) [inline src/interface_adapter/login/LoginPresenter.java:1]: Currently no failview in place!

**Review by hayatariq — APPROVED** [review]

**Comment by chowdberry** [comment]
> Please include getPromptByID and getResponses! would help me out, thanks

Implemented both of these methods, unit testing should be done to confirm these implementations are working properly.

**Pull request merged**